apps/wing/project-templates/wing/react-vite/backend/main.w

```diff
@@ -0,0 +1,47 @@
+bring cloud;
+bring expect;
+bring vite;
+bring http;
+bring "./broadcaster.w" as b;
+
+// Winglang doesn't have a built-in support for __dirname yet, so we use a workaround to get the current directory.
+// @see tracking issue: https://github.com/winglang/wing/issues/3736
+class Utils {
+  extern "utils.js" pub static __dirname(): str;
+}
+
+let my_broadcaster = new b.Broadcaster() as "Broadcaster";
```

Comment:
I usually try to avoid big minifications like `bring as b` because I find them a bit confusing. Also, the `as "Broadcaster" feels unnecessary. What about:

```wing
bring "./broadcaster.w" as broadcaster;
let myBroadcaster = new broadcaster.Broadcaster();
```

And also:

```wing
let website = new vite.Vite(
```

Reply:
updated